packages/api/src/utils.js

```diff
@@ -15,6 +15,20 @@ const getKnex = () => {
   })
 }
 
+const calculateEpoch = ({ index, genesisTime, currentBlock }) => {
```

Comment:
make regular arguments instead of object

Reply:
also replace currentBlock with timestamp

Reply:
replace currentBlock with timestamp ✅